lib/root.js

```diff
@@ -15,6 +15,14 @@ process.on('exit', function (code) {
     process.exit(1)
 })
 
+// need to autoend if a teardown is added.
+// otherwise we may never see process.on('exit')!
+tap.tearDown = tap.afterAll = function (fn) {
+  var ret = Test.prototype.afterAll.apply(this, arguments)
+  this.autoend()
```

Comment:
Root-level `tap.tearDown()` is a must-have unfortunately.  The lack of it is keeping many would-be tap users on 0.x